docs/GraphNode.fbmd

```diff
@@ -270,111 +270,116 @@ Returns the `perms` property for the page as an `array` if present. (Only availa
 <card>
 ## GraphAlbum Instance Methods {#album-instance-methods}
 
-The `GraphAlbum` collection represents an [Album](https://developers.facebook.com/docs/graph-api/reference/album) Graph node.
```

Comment:
Is there a reason why you removed this line? I think we should keep the reference to the Graph API docs for this node.